pandas/core/window/ewm.py

```diff
@@ -160,7 +160,7 @@ def _constructor(self):
         """
     See Also
     --------
-    pandas.DataFrame.rolling.aggregate
+    pandas.DataFrame.rolling.aggregate: Aggregate using one or more operations over the specified axis.
```

Comment:
IMO ending with " over the specified axis" makes the sentence heavy to read. I think it would say the same if that ending was just dropped?

Same goes for the doc strings below.